monitoring-datastores/snowflake/main.tf

```diff
@@ -0,0 +1,55 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "monitor_snowflake_datastore_lambda_function" {
+  source = "terraform-aws-modules/lambda/aws"
+  
+  function_name = "monitor-snowflake-datastore-lambda"
+  description   = "Lambda function that monitors snowflake datastore"
+  handler       = "monitor-snowflake-datastore.main"
+  runtime       = "python3.10"
+
+  publish                  = true
+
+  source_path = "${path.module}/src"
+  build_in_docker  = true
+
+  timeout = 60
+
+  environment_variables = {
+    snowflake_user = "",
+    snowflake_password = "",
+    snowflake_account = "",
+    snowflake_satori_host = "",
+    snowflake_warehouse = ""
```

Comment:
You don't want to pass it as variables and explain to a customer how to pass it in secured way ?